docs/docs/cli.md

```diff
@@ -1,91 +1,204 @@
 
-This option specifies the `PATH` to be used for temporary storage during
-prediction. The model that is shipped with `zamba` is able to process within
-memory (assuming reasonably large modern memory of ~16 GB). If a custom model
-is being used, it may be necessary to point `zamba` to a mounted drive or some
- other large-capacity directory. By default this uses the operating system's temporary directory.
+#### `--batch-size INT`
 
-#### --proba_threshold FLOAT
+The batch size to use for prediction.
 
-For advanced uses, you may want the algorithm to be more or less sensitive to if a species is present. This parameter is a `FLOAT` number, e.g., `0.64` corresponding to the probability
-threshold beyond which an animal is considered to be present in the video being
- analyzed.
+#### `--save`
 
-By default no threshold is passed, `proba_threshold=None`. This will return a probability from 0-1 for each species that could occur in each video. If a threshold is passed,
-then the final prediction value returned for each class is `probability >= proba_threshold`, so that all class values become `0` (`False`, the species does not appear) or `1` (`True`, the species does appear).
+Whether to save out the predictions to a CSV file. Predictions will be saved by default. Specify `--no-save` if you would like to write out predictions.
 
-#### --output_class_names
+#### `--dry-run`
 
-Setting this option to `True` yields the most concise output `zamba` is capable
- of. The highest species probability in a video is taken to be the _only_
- species in that video, and the output returned is simply the video name and
-  the name of the species (or `blank`) with the highest class probability. See
-  the [Quickstart](index.md) for example usage.
+Specifying `--dry-run` is useful for trying out model implementations more quickly by running only a single batch of inference. The default is `--no-dry-run`.
 
-#### --model_profile TEXT
+#### `--config`
 
-There are two versions of the algorithm that ship with zamba. If you pass `fast` there is a faster algorithm that can be less accurate that is used. If you pass `full` (the default) a slower algorithm that has 4 sub-models instead of 1 is used.
+Path to a yaml configuration file with values for other arguments to `predict`. If a value is specified in both the command line and in a yaml file that is passed, the command line argument value will be used. Each default model (`time_distributed`, `slowfast`, and `european`) comes with a yaml file that sets default configurations. If `--config` is not specified, these default values will be used for any argument that is not passed directly to the command line. Default configs can be found in `models/configs`.<!-- TODO: make sure that's right><!-->
 
```

Comment:
Confirm that `models/config` is where we'll store the default configuration for each model

Reply:
Probably `zamba/models/configs`, but this is the kind of thing where we will want this to be a direct link to that folder on GitHub so people don't have to go poking around.